### PR TITLE
Added new packages visualization and toolkit

### PR DESCRIPTION
**Pull request by BenHakimIlyass**
ALL CONTRIBUTIONS WILL BE REVIEWED BY AT LEAST ONE OF OUR FRONTEND TEAM MEMBERS.

## Technical description

Created new packages, visualization and toolkit.
- Visualization containing DamageAnnotations and DamageHighlights
- Toolkit containing utils, our custom proptypes and common hooks

*This Pull Request template has been written and generated by Monk JS repository.*

### CODE REVIEW

**Comment by BenHakimIlyass** [comment]
In this PR we don't remove react-native and react-native-views packages, which means next to do is to make sure that we no longer need them and then deprecate them with ease.

File: packages/toolkit/package.json [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    "README.md"
+  ],
+  "scripts": {
+    "watch": "watch 'yarn run build' src",
+    "lint": "eslint src --cache",
+    "build": "bob build",
+    "release": "release-it"
+  },
+  "dependencies": {
+    "react-native-paper": "^4.10.1",
```
Comment (natsukiai) [inline packages/toolkit/package.json:42]: Do we use RNP is this package ?

**Review by natsukiai — COMMENTED** [review]
I've reviewed it but there is some refactor to do with toolkit. I'll migrate camera first like you say.

File: packages/toolkit/package.json (hunk 1 above)
Comment (BenHakimIlyass) [reply packages/toolkit/package.json:42]: Yes we use it inside DamageAnnotation

**Pull request merged**